janus_core/calculations/phonons.py

```diff
@@ -149,6 +150,7 @@ def __init__(
         write_results: bool = True,
         write_full: bool = True,
         file_prefix: Optional[PathLike] = None,
+        enable_progress_bar: bool = False
```

Comment:
```suggestion
        enable_progress_bar: bool = False,
```
To make ruff happy (best merged with other suggestion)